lib/tasks/recreate_and_seed.rake

```diff
@@ -0,0 +1,23 @@
+require './app/importers/invoices_importer'
+
+desc 'Drop and recreate database AND import all data from CSVs'
+namespace :import do
+  task :all => :environment do
+    Rake::Task["db:drop"].invoke
+    Rake::Task["db:create"].invoke
+    Rake::Task["db:schema:load"].invoke
+    customers =     './db/csv/customers.csv'
+    merchants =     './db/csv/merchants.csv'
+    items =         './db/csv/items.csv'
+    invoices =      './db/csv/invoices.csv'
+    transactions =  './db/csv/transactions.csv'
+    invoice_items = './db/csv/invoice_items.csv'
+    puts "\n Yes, you have time to grab a cup of coffee. \n If you leave now.\n\n"
+    CustomersImporter.new(customers).import
+    MerchantsImporter.new(merchants).import
+    ItemsImporter.new(items).import
+    InvoicesImporter.new(invoices).import
+    TransactionsImporter.new(transactions).import
+    InvoiceItemsImporter.new(invoice_items).import
+  end
+end
```

Comment:
I love this rake task! Since the order is specific I think it's a great idea. Also it will save me from having to type it all in every time we change the schema. Wahoo. 👍